grakn-engine/src/main/java/ai/grakn/engine/postprocessing/PostProcessing.java

```diff
@@ -51,22 +49,17 @@
  */
 public class PostProcessing {
```

Comment:
We dont really need this class at all now - can eventually move the whole thing into `PostProcessingTask`